src/synthesis.ts

```diff
@@ -1,38 +1,56 @@
 import {
   availableAmount,
   Effect,
+  mallPrice,
   mySpleenUse,
   retrieveItem,
   spleenLimit,
   sweetSynthesis,
   sweetSynthesisResult,
 } from "kolmafia";
-import { $items, clamp } from "libram";
+import { $item, $items, clamp } from "libram";
 import { shuffle } from "./lib";
 
-const whitelist = $items`sugar shotgun, sugar shillelagh, sugar shank, sugar chapeau, sugar shorts, sugar shield, sugar shirt, Fudgie Roll`;
+const allowList = $items`Fudgie Roll, peanut brittle shield, sugar shotgun, sugar shillelagh, sugar shank, sugar chapeau, sugar shorts, sugar shield, sugar shirt`;
+
+// For safety, explicitly skip candies that are no longer obtainable or extremely rare
+const blockList = new Set([
+  $item`candied nuts`,
+  $item`candy kneecapping stick`,
+  $item`chocolate cigar`,
+  $item`fancy but probably evil chocolate`,
+  $item`fancy chocolate`,
+  $item`fancy chocolate car`,
+  $item`gummi ammonite`,
+  $item`gummi belemnite`,
+  $item`gummi trilobite`,
+  $item`powdered candy sushi set`,
+  $item`radio button candy`,
+  $item`spiritual candy cane`,
+  $item`Ultra Mega Sour Ball`,
+  $item`vitachoconutriment capsule`,
+]);
```

Comment:
Also: I think "blockList" is much better than the "skipList" we ended up with when I was just blindly replacing blacklist; we should let that trickle around everywhere

Reply:
> We should let eslint-plugin-libram and libram's template literals let us do line breaks and indentation. Or would IDEs not indent smartly?

That is the dream. I think it would indent fine if supported.